doc/dev-manual-cn.md

```diff
@@ -98,4 +98,5 @@ Makefile rules:
 1. 先安装 perl 的包管理器 cpanminus
 2. 然后通过 cpanm 来安装 test-gninx：`sudo cpanm --notest Test::Nginx IPC::Run > build.log 2>&1 || (cat build.log && exit 1)`
 3. 然后 clone 最新的源码：`git clone https://github.com/openresty/test-nginx.git`
-4. 通过 perl 的 `prove` 命令来加载 test-nginx 的库，并运行 `/t` 目录下的测试案例集:`prove -Itest-nginx/lib -r t`
+4. 通过 perl 的 `prove` 命令来加载 test-nginx 的库，并运行 `/t` 目录下的测试案例集：`prove -Itest-nginx/lib -r t`
+>可通过 TEST_NGINX_BINARY 环境变量来设置执行的 nginx 路径，如 TEST_NGINX_BINARY=/usr/local/bin/openresty prove -Itest-nginx/lib -r t
```

Comment:
how about this style? I think it is more clear.

```
... ...
3. xxxx
4. ****, 并运行 `/t` 目录下的测试案例集:
    * 直接运行：`prove -Itest-nginx/lib -r t`
    * 指定 Ningx 二进制路径：`TEST_NGINX_BINARY=/usr/local/bin/openresty prove -Itest-nginx/lib -r t`
```

Reply:
nice suggestion,if we have other recommendations,we can also list them in this way.